### PR TITLE
Make BikeAnnotationView more accessible

### PR DESCRIPTION
**Pull request by besilva**
## 📝 Docs

- [x] Did you update the [CHANGELOG.md](https://github.com/criticalmaps/criticalmaps-ios/blob/master/CHANGELOG.md)?

## 📲 What

Now Bike Annotation scales with  `preferredContentSizeCategory`

## 🤔 Why

This way the view is easier to see for people who use their device with larger texts
Also I changed the way the view is created because it was creating a Optimization Opportunity like in the image below
Screen Shot 2020-11-14 at 03 15 22

## 👀 See

Screenshots, external resources?
- [x] Added snapshot tests (using iPhone 8 Simulator)

| Before 🐛 | After 🦋 |
| --- | --- |
| default IMG_1998 |  IMG_1998 |
| extra large IMG_1998 | IMG_1999 |
| large IMG_1998 |  IMG_2001 |
| small IMG_1998   | IMG_2002 |

## ♿️ Accessibility 

- [x] Tap targets use minimum of 44x44 pts dimensions
- [x] Works with VoiceOver
- [x] Supports Dynamic Type

### CODE REVIEW

**Comment by besilva** [comment]
#242  I think the design should give some feedback about the sizes and also I didn't know what to put on Changelog.

**Review by mltbnz — APPROVED** [review]
This is cool. 
I thought it would be more complicated but the implementation looks fairly straight forward now that I see it. 

Also I think the sizes are good. We could maybe iterate on that and maybe someone gives some feedback

**Comment by mltbnz** [comment]
Also really cool you fixed the optimization opportunity! 🚀

**Comment by besilva** [comment]
@mltbnz Thanks a lot! I think the idea of use the feedback to improve this feature is perfect.

**Comment by besilva** [comment]
Update Changelog but I'm not sure if it was in the right place hehe

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Changelog for Critical Maps iOS
 - Fix missing information for VoiceOver user for the navigationbar buttons
 - Fix VoiceControl labels on each view
 - Fix `bottomContentOffset` for NavigationOverlay (did hide the Legal button)
+- Fix `BikeAnnotation` it is more accesible now(scales with preferredContentSizeCategory)
```
Comment (fbernutz) [inline CHANGELOG.md:23]: Sorry for that, I added a new `Unreleased` section on top of the changelog. Could you please merge the latest change from  `master` and move your line on top to the unreleased section? :)

Comment (fbernutz) [reply CHANGELOG.md:23]: Just a minor suggestion:

```suggestion
- Make `BikeAnnotation` more accessible (scales with preferredContentSizeCategory)
```

Comment (besilva) [reply CHANGELOG.md]: I agree with you, just made the changes

**Review by fbernutz — APPROVED** [review]

**Pull request merged**